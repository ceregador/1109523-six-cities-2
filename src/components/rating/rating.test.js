import React from 'react';
import Rating from '../rating/rating.jsx';
import Renderer from 'react-test-renderer';

it(`renders correctly`, () => {
  const tree = Renderer
    .create(<Rating value={2.5}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
