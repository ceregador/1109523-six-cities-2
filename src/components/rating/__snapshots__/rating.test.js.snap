// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="place-card__rating rating"
>
  <div
    className="place-card__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "60%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;
